libs/application/templates/driving-license/src/lib/messages.ts

```diff
@@ -1,9 +1,319 @@
 import { defineMessages } from 'react-intl'
 
 export const m = defineMessages({
-  testing: {
-    id: 'dl.application:bikeType.A2',
-    defaultMessage: '<span><b>A2</b> Bifhjól</span>',
-    description: '<span><b>A2</b> Bike</span>',
+  yes: {
+    id: 'dl.application:shared.yes',
+    defaultMessage: 'Já',
+    description: 'Yes',
+  },
+  no: {
+    id: 'dl.application:shared.no',
+    defaultMessage: 'Nei',
+    description: 'No',
+  },
+  applicationName: {
+    id: 'dl.application:application.name',
+    defaultMessage: 'Ökuskírteini',
+    description: 'Driving license',
+  },
+  externalDataSection: {
+    id: 'dl.application:externalData.section',
+    defaultMessage: 'Ökuréttindi',
+    description: 'Driving license',
+  },
+  externalDataTitle: {
+    id: 'dl.application:externalData.title',
+    defaultMessage: 'Upplýsingasöfnun og skilyrði',
+    description: 'Information gathering and conditions',
+  },
+  nationalRegistryTitle: {
+    id: 'dl.application:nationalRegistry.title',
+    defaultMessage: 'Persónuupplýsingar úr Þjóðskrá',
+    description: 'Personal information from the National Registry',
+  },
+  nationalRegistrySubTitle: {
+    id: 'dl.application:nationalRegistry.subTitle',
+    defaultMessage:
+      'Til þess að auðvelda fyrir sækjum við persónuupplýsingar úr Þjóðskrá til þess að fylla út í umsóknina',
+    description:
+      'Information from the National Registry will be used to prefill the data in the application',
+  },
+  userProfileInformationTitle: {
+    id: 'pl.application:userprofile.title',
+    defaultMessage: 'Netfang og símanúmer úr þínum stillingum',
+    description: 'Your user profile information',
+  },
+  userProfileInformationSubTitle: {
+    id: 'pl.application:userprofile.subTitle',
+    defaultMessage:
+      'Til þess að auðvelda umsóknarferlið er gott að hafa stillt netfang og símanúmer á mínum síðum',
+    description:
+      'In order to apply for this application we need your email and phone number',
+  },
+  penaltyPointsTitle: {
+    id: 'pl.application:penaltyPoints.title',
+    defaultMessage: 'Punktastaða úr Ökuskírteinaskrá',
+    description: 'Penalty points from Ökuskírteinaskrá',
+  },
+  penaltyPointsSubTitle: {
+    id: 'pl.application:penaltyPoints.subTitle',
+    defaultMessage:
+      'Til þess að tryggja að notandi hafi heimild til þess að sækja um ökuskírteini út frá punktastöðu',
+    description:
+      'In order to ensure the user has permission to apply for driving license',
```

Comment:
`In order to ensure the user's penalty points do not prohibit him from applying for a driving license`